app/src/main/java/com/geeksville/mesh/NodeInfo.kt

```diff
@@ -219,6 +220,8 @@ data class NodeInfo(
     var snr: Float = Float.MAX_VALUE,
     var rssi: Int = Int.MAX_VALUE,
     var lastHeard: Int = 0, // the last time we've seen this node in secs since 1970
+    @ColumnInfo(name="hopsAway", defaultValue = "-1")
+    var hopsAway: Int = -1,
```

Comment:
I've now moved hopsAway to the end.

I'm having issues removing `@ColumnInfo` though, when I remove that line it will not build, I receive the error `"New NOT NULL column'hopsAway' added with no default value specified. Please specify the default value using @ColumnInfo."`. I'm not exactly sure why it is required compared to other columns, but looking online it's related to versioning of the Room database.